oppia-tools-dependency-list.txt

```diff
@@ -0,0 +1,8 @@
+google_appengine_1.9.73
```

Comment:
What's the process for ensuring that this file is up-to-date (and that we don't have a dev who adds/removes/modifies a dep but forgets to change this file)?

Reply:
Instead of having a static file like this, I can run a Shell script that generates this text file listing all directory names within `/oppia_tools` and saves it somewhere, let's say `/temp`. I can then have CircleCI `checksum /temp/oppia-tools-dependency-list.txt`.

Would that be better here? What do you think?

Reply:
That seems reasonable to me, and more robust. An alternative is to keep the text file in the repo but have a check beforehand to ensure that the text file and the shell script output match each other.